theano/gpuarray/dnn.py

```diff
@@ -3054,7 +3054,7 @@ def local_abstractconv_cudnn_graph(op, context_name, inputs, outputs):
                         conv_mode=conv_mode,
                         num_groups=op.num_groups)
     elif isinstance(op, AbstractConv2d_gradWeights):
-        shape = (inp2.shape[1], inp1.shape[1],
+        shape = (inp2.shape[1], inp1.shape[1] // op.num_groups,
```

Comment:
It happen to everybody. I like to make the distinction between a few catétory of bug depending of how it affect user. Mostly, is it a user visible bug or not.

If it is a crash, it is not the worst bug, as user see it clearly, it is not too bad.

The worst bug are if we return wrong result for some reasons.